roles/openshift_examples/examples-sync.sh

```diff
@@ -45,8 +45,8 @@ mv origin-${ORIGIN_BRANCH}/examples/quickstarts/*.{yaml,json} ${EXAMPLES_BASE}/q
 mv origin-${ORIGIN_BRANCH}/examples/jenkins/jenkins-*template.json ${EXAMPLES_BASE}/quickstart-templates/
 mv origin-${ORIGIN_BRANCH}/examples/image-streams/*.{yaml,json} ${EXAMPLES_BASE}/image-streams/
 mv application-templates-${XPAAS_VERSION}/jboss-image-streams.json ${EXAMPLES_BASE}/xpaas-streams/
-mv rhdm-7-openshift-image-${RHDM_7_TAG}/rhdm72-image-streams.yaml ${EXAMPLES_BASE}/xpaas-streams/
-mv rhpam-7-openshift-image-${RHPAM_7_TAG}/rhpam72-image-streams.yaml ${EXAMPLES_BASE}/xpaas-streams/
```

Comment:
Since we don't sync those anymore should 7.2 be removed from the streams here?

Reply:
I didn't understand your question.
Does it means that the image streams imported here will not be used anymore and it will be read from origin/library?

Reply:
Yeah, removing these lines would stop these imagestreams from syncing. Do we still need those in the examples then?

Reply:
Ah ok, yes, we deprecate older images in favor of new ones, in this case the images streams will be replaced by the 7.3.

Reply:
Ah, got it, thanks. Lets avoid removing things then